src/js/src/components/SearchFacetOptions.vue

```diff
@@ -46,6 +48,19 @@ export default {
       this.updateSolrSettingOffset(0)
       this.addToSearchAppliedFacets(newFacet)
       this.$_pushSearchHistory('Search', this.query, this.searchAppliedFacets, this.solrSettings)
+    },
+    checkForFacets(facets) {
+    //we test if the variable exists first - can cause problems if it's not set yet.
+    if(facets) {
+      let fate = false
+      Object.keys(facets).forEach(function(item) {
+        facets[item].length !== 0 ? fate = true : null
+      })
+      return fate
+      }
+      else {
+        return false
+      }
```

Comment:
This can be optimised: `
 let fate = false      
if(facets) {
      Object.keys(facets).forEach(function(item) {
        facets[item].length !== 0 ? fate = true : null
      })
     }
       return fate
 `

Reply:
Don't now where this ended but the branch is good to go so if you wan't to save the suggested improvement for later give me a ping and I will approve and merge in.